src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int DEFAULT_VALUE = 10;
+    private static final Color COLOR_WHITE = Color.WHITE;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = getRandomNumber();
+        String color = colorSupplier.getRandomColor();
+
+        switch (figureNumber) {
+            case 0:
+                return new Square(color, getRandomNumber());
+            case 1:
+                return new Rectangle(color, getRandomNumber(), getRandomNumber());
+            case 2:
+                return new RightTriangle(color, getRandomNumber(), getRandomNumber());
+            case 3:
+                return new Circle(color, getRandomNumber());
+            case 4:
+                return new IsoscelesTrapezoid(color,
+                            getRandomNumber(),
+                            getRandomNumber(),
+                            getRandomNumber());
+            default:
+                return new IsoscelesTrapezoid(color,
+                            getRandomNumber(),
+                            getRandomNumber(),
+                            getRandomNumber());
+        }
```

Comment:
In this case you need to return default figure - white circle with a radius of 10